library/src/main/cpp/CMakeLists.txt

```diff
@@ -43,11 +43,12 @@ if(WITH_HEIF OR WITH_AVIF)
   add_definitions(-DHAVE_LIBHEIF)
   target_sources(imagedecoder PRIVATE decoder_heif.cpp)
 endif()
+
+add_subdirectory(libjxl)
```

Comment:
Why was this pulled out?

Reply:
It's to use lcms bundled with libjxl. If it's desired, I can revert it since the work's been done to use it separately.

Reply:
Yeah, I agree. All the other decoders are optional so making this one explicit defeats the whole purpose.